### DISCUSSION TITLE
Portuguese Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#392] [Language Leaders]
@marcocunha, @joaorgoncalves, @Alexivia - is anyone interested in being a language leader for Portuguese?

**Comment by joaorgoncalves** [comment]
Hey @synesthesiam, I'm interested. Let's do it!

**Reply by marcocunha** [reply]
Great @joaorgoncalves that you're taking the lead. I'll also try to contribute as much as possible

**Reply by Alexivia** [reply]
Good João! I will also be doing some contributions when possible 😃

**Reply by joaorgoncalves** [reply]
All help is welcome 😄 Thanks

**Reply by emanuelsilvapt** [reply]
Any additional help needed? I would like to help too

**Comment by synesthesiam** [comment]
Thanks @joaorgoncalves! Adding you as a language leader.

**Reply by nikito7** [reply]
@synesthesiam can I be leader too?

**Comment by synesthesiam** [comment]
@joaorgoncalves Would you or others be able to translate the new responses files into Portuguese?

**Reply by joaorgoncalves** [reply]
@synesthesiam Yes, I can translate, no problem. I'm going to take a look into that in the next few days.

**Comment by synesthesiam** [comment]
@joaorgoncalves when you get a chance, could you please look into translating the `HassGetState` sentences/responses?

**Reply by nikito7** [reply]
@joaorgoncalves are you alive?

**Reply by synesthesiam** [reply]
I hope so! If they're busy, that's fine. I will add you as another language leader @nikito7

**Reply by nikito7** [reply]
Thank you

**Reply by Alexivia** [reply]
This weekend I might also have some time to work on this.

**Comment by nikito7** [comment]
I'm away. Portuguese language needs a new leader.

**Reply by ViPeR5000** [reply]
:(

**Comment by synesthesiam** [comment]
Any volunteers for a language leader?

**Reply by ViPeR5000** [reply]
I can try help.

**Comment by ViPeR5000** [comment]
@synesthesiam I am available

**Comment by joaorgoncalves** [comment]
Hey guys. Sorry for disappearing. My daughter was born and my available time for side projects changed a lot 😬🤣 I'll be around whenever I have time but I won't be a good language leader at the moment. Sorry, and thanks for the opportunity 😞 
Ps: I'll still be using home assistant and contributing as possible.

**Reply by ViPeR5000** [reply]
congratulations

**Comment by synesthesiam** [comment]
@ViPeR5000 do you have any PR's I can take a look at? We usually ask language leaders to have submitted at least 2 small ones.

**Comment by synesthesiam** [comment]
@abmantis Pinging you here about being a language leader

**Reply by abmantis** [reply]
I'm in!

**Reply by synesthesiam** [reply]
Great, added you as a language leader! It may take a bit on the Github side before you are able to merge.

**Reply by abmantis** [reply]
Thanks!

**Reply by synesthesiam** [reply]
@abmantis You should be able to moderate Portuguese PR's now
